multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -27,6 +27,11 @@
 
 from autogluon.common.utils.log_utils import set_logger_verbosity, verbosity2loglevel
 from autogluon.multimodal.utils import save_result_df
+from autogluon.multimodal.models.fusion import AbstractMultimodalFusionModel
+from autogluon.multimodal.models.huggingface_text import HFAutoModelForTextPrediction
+from autogluon.multimodal.models.mmdet_image import MMDetAutoModelForObjectDetection
+from autogluon.multimodal.models.timm_image import TimmAutoModelForImagePrediction
```

Comment:
It seems that these model classes are not used.

Reply:
Yup they are deleted now